packages/helpers/qrcode-modal/package.json

```
@@ -9,7 +9,7 @@
     "build:pre": "run-s clean build:svg build:css",
     "build:umd": "webpack",
     "build:cjs": "microbundle --no-compress -f cjs --alias react=preact/compat,react-dom/test-utils=preact/test-utils,react-dom=preact/compat --tsconfig ./tsconfig.cjs.json",
-    "build": "run-s build:pre build:cjs build:umd",
+    "build": "run-s build:pre build:cjs",
```

We removed this to unblock the builds for the other modules, since the UMD build here was having cryptic TS issues.

@pedrouid we need to add this back before merging to avoid quietly discontinuing UMD builds:

```suggestion
    "build": "run-s build:pre build:cjs build:umd",
```